README.md

```diff
@@ -75,3 +81,7 @@ snakemake -j 8
 ```
 
 ...and wait a day or two for results, or a crash that may tell you what you need to fix.
+
+To build `parallel` corpus, run `./build.sh parallel`. If you want `alignment` files for
```

Comment:
Just a minor typographical nitpick: I think text in backticks `like this` should only be used for text that is meant to be code, or examples of input or output. If it's just a word in a sentence and you want to emphasize it, use underscores.

Reply:
Got it. Fixed.